operator/internal/securedcluster/values/defaults/defaults.go

```diff
@@ -29,6 +29,7 @@ var staticDefaults = platform.SecuredClusterSpec{
 		HostAliases:     nil,
 	},
 	AuditLogs:        nil,
+	ProcessBaselines: nil,
```

Comment:
If you rebase, you'll find none of the siblings are `nil` any more (they were temporarily between merging https://github.com/stackrox/stackrox/pull/16193 and https://github.com/stackrox/stackrox/pull/16305), please also fill in the value for your field.

Reply:
I have rebased and given `ProcessBaselines` a default value. It is disabled by default.